projections/dnfmit/SECURITY.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Should I bump the version of these files in this PR, or do they get bumped whenever "a release" is cut?

Reply:
If you could bump the comment at the top of the Markdown files, that'd help!